### DISCUSSION TITLE
How to Implement Silent Printing in Tauri?

### DISCUSSION THREAD

**Discussion by randiekas** [tauri-apps/tauri#7832] [Q&A]
I'm looking to initiate silent printing functionality within a Tauri application, and I could use some guidance on how to achieve this. Silent printing allows for automated printing without user interaction, which can be useful in various scenarios.

I'm particularly interested in learning about the best practices, libraries, or methods to implement silent printing within a Tauri app. If you have experience or insights on this topic, please share your expertise and suggestions in this discussion. Your input would be greatly appreciated!

**Comment by FabianLars** [comment]
For windows there's a community plugin here: https://github.com/alfianlensundev/tauri-plugin-printer

In general though, the feature request for an official implementation is tracked here: https://github.com/tauri-apps/plugins-workspace/issues/293
